04 Strategy Library/102 Option Expiration Week Effect/02 Method.html

```diff
@@ -0,0 +1,50 @@
+<p>
+  S&amp;P 100 index includes 102 leading U.S. stocks with exchange-listed options. The constituents represent almost 51% of the market capitalization of the U.S. equity market.
+  Here we trade the S&amp;P 100 index ETF as a portfolio of U.S. large-cap stocks.
+</p>
+<p>
+  In the next step, we add the options to get their expiration dates. The range of expiration dates should bigger enough to include the
+  the contract which expires in this month.
+</P>
+<div class="section-example-container">
+<pre class="python">
+def Initialize(self):
+    self.SetStartDate(2007, 1, 1)
+    self.SetEndDate(2018, 8, 1)
+    self.SetCash(10000)
+    self.AddEquity("OEF", Resolution.Minute)
+    option = self.AddOption("OEF")
+    option.SetFilter(-3, 3, timedelta(0), timedelta(days = 60))
+</pre>
```

Comment:
Would be nice to add SetBenchmark("OEF") here to compare it to the underlying index.